compare-bug-version-and-git-branch.sh

```diff
@@ -81,8 +81,12 @@ BUG_VER=$(cut -d. -f1,2  <<< "${BUG_VERSION}")
 GERRIT_VER=$(sed 's/.*-//' <<< "${GERRIT_BRANCH}")
```

Comment:
Could we cut out the second field from GERRIT_BRANCH here

GERRIT_VER=$(cut -f 2 -d '-' <<< "${GERRIT_BRANCH}")

This will keep changes minimal and accommodate any more branches with that naming pattern and corresponding bug version.